cluster-autoscaler/utils/kubernetes/factory.go

```diff
@@ -41,9 +39,13 @@ const (
 // CreateEventRecorder creates an event recorder to send custom events to Kubernetes to be recorded for targeted Kubernetes objects
 func CreateEventRecorder(kubeClient clientset.Interface) kube_record.EventRecorder {
 	eventBroadcaster := kube_record.NewBroadcasterWithCorrelatorOptions(getCorrelationOptions())
-	eventBroadcaster.StartLogging(klog.V(4).Infof)
 	if _, isfake := kubeClient.(*fake.Clientset); !isfake {
-		eventBroadcaster.StartRecordingToSink(&v1core.EventSinkImpl{Interface: v1core.New(kubeClient.CoreV1().RESTClient()).Events("")})
+		actualSink := &v1core.EventSinkImpl{Interface: v1core.New(kubeClient.CoreV1().RESTClient()).Events("")}
+		// EventBroadcaster has a StartLogging() method but the throttling options from getCorrelationOptions() get applied only to
```

Comment:
Is there an issue created to fix the root cause? 
Can you add a `//TODO` here and reference the issue?

Reply:
Added!